java/src/main/java/com/amido/stacks/workloads/menu/api/v1/DeleteItemController.java

```diff
@@ -1,68 +1,41 @@
 import io.swagger.v3.oas.annotations.media.Content;
 import io.swagger.v3.oas.annotations.media.Schema;
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
-import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import java.util.UUID;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestAttribute;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 
-/** @author ArathyKrishna */
-@RequestMapping("/v1/menu/{id}/category/{categoryId}/items/{itemId}")
+@RequestMapping(
+    path = "/v1/menu/{id}/category/{categoryId}/items/{itemId}",
+    produces = MediaType.APPLICATION_JSON_VALUE + "; charset=utf-8",
+    method = RequestMethod.DELETE)
 public interface DeleteItemController {
 
-  @DeleteMapping(produces = "application/json; charset=utf-8")
+  @DeleteMapping
   @Operation(
       tags = "Item",
       summary = "Removes an item from menu",
-      security = @SecurityRequirement(name = "bearerAuth"),
       description = "Removes an item from menu",
-      operationId = "DeleteMenuItem",
-      responses = {
-        @ApiResponse(
-            responseCode = "200",
-            description = "Success",
-            content = @Content(mediaType = "application/json", schema = @Schema(hidden = true))),
-        @ApiResponse(
-            responseCode = "204",
-            description = "No Content",
-            content = @Content(schema = @Schema(hidden = true))),
-        @ApiResponse(
-            responseCode = "400",
-            description = "Bad Request",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "401",
-            description = "Unauthorized, Access token is missing or invalid",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "403",
-            description = "Forbidden, the user does not have permission to execute this operation",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "404",
-            description = "Resource not found",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class)))
-      })
+      operationId = "DeleteMenuItem")
+  @ApiResponse(
```

Comment:
Any reason why this 404 could not be put on the base annotation (@DeleteAPIResponses)?